geonode/resource/manager.py

```diff
@@ -341,7 +343,7 @@ def create(self, uuid: str, /, resource_type: typing.Optional[object] = None, de
                         uuid, resource_type=resource_type, defaults=resource_dict
                     )
                 _resource.save()
-                resourcebase_post_save(_resource.get_real_instance())
+                resourcebase_post_save(_resource.get_real_instance(), **kwargs)
```

Comment:
Since we are changing this, i guess could be worth to pass both `args` and `kwargs` waht you think @ridoo?

Reply:
yes .. no problem.

@mattiagiupponi could you have a look at the failing test? Actually, the test run ok locally. I do not have a clue why it is failing on circleci. If it fails on your machine as well, I would have to re-check my setup.

Reply:
Are your refering to the clone test?

```
ERROR: test_clone_remote_files_remote (geonode.storage.tests.TestDataRetriever)
ERROR: test_get_arcgis_alternative_structure (geonode.services.tests.ModuleFunctionsTestCase)
FAIL: test_create_passing_custom_to_post_save (geonode.resource.tests.TestResourceManager)
```